frontend/test/metabase/scenarios/joins/joins.cy.spec.js

```diff
@@ -443,7 +443,7 @@ describe("scenarios > question > joined questions", () => {
     cy.get(".ScalarValue").contains("2,087");
   });
 
-  it("should show 'Previous results' instead of a table name for non-field dimensions (metabase#17968)", () => {
+  it.skip("should show 'Previous results' instead of a table name for non-field dimensions (metabase#17968)", () => {
```

Comment:
@kulyk -- This scenario now returns a Table, which seems correct, I think, but perhaps I am wrong here. I couldn't find a scenario where the table or something is missing, causing the `"Previous results"` string to show up.

Reply:
![Screen Shot 2022-09-08 at 10 41 45 AM](https://user-images.githubusercontent.com/13057258/189151917-d9103fd4-de80-40dc-8549-ffd83ee6575b.png)

Repro'd it this way -- with custom columns. Does this behavior make sense?

Reply:
Hrm, not so sure. @flamber mind giving this a sanity check?

Reply:
I guess you're looking for something like this
![image](https://user-images.githubusercontent.com/1447303/189159085-3c8dfb2e-e17c-40f9-9449-aa53a496b535.png)

Reply:
Thanks, found the problem -- https://github.com/metabase/metabase/pull/25109/commits/de9eab32a8cfb7a65252f516964b31a4f2154f7b